packages/node/src/handlers.ts

```diff
@@ -65,6 +75,7 @@ export function tracingHandler(): (
     res.once('finish', () => {
       // We schedule the immediate execution of the `finish` to let all the spans being closed first.
```

Comment:
```suggestion
      // Push `transaction.finish` to the next event loop so open spans have a chance to finish before the transaction closes
```

^^^ Please tell me if I've gotten the meaning right. I know that it's `setImmediate` but saying "we schedule it to happen immediately" feels misleading, since the whole point of what we're doing is pushing it off farther into the future.

Reply:
Thanks